jbpm-console-ng-human-tasks/jbpm-console-ng-human-tasks-client/src/test/java/org/jbpm/console/ng/ht/client/editors/taskslist/grid/dash/DataSetTaskListGridPresenterTest.java

```diff
@@ -69,15 +89,63 @@
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_DESCRIPTION, 0)).thenReturn("taskDescription");
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_STATUS, 0)).thenReturn("1");
+        when(dataSetQueryHelperMock.getColumnIntValue(dataSetMock, COLUMN_PRIORITY, 0)).thenReturn(1);
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_ACTUALOWNER, 0)).thenReturn("user");
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_CREATEDBY, 0)).thenReturn("user");
+        when(dataSetQueryHelperMock.getColumnDateValue(dataSetMock, COLUMN_CREATEDON, 0)).thenReturn(new Date());
+        when(dataSetQueryHelperMock.getColumnDateValue(dataSetMock, COLUMN_ACTIVATIONTIME, 0)).thenReturn(new Date());
+        when(dataSetQueryHelperMock.getColumnDateValue(dataSetMock, COLUMN_DUEDATE, 0)).thenReturn(new Date());
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_PROCESSID, 0)).thenReturn("testProc");
+        when(dataSetQueryHelperMock.getColumnLongValue(dataSetMock, COLUMN_PROCESSSESSIONID, 0)).thenReturn(Long.valueOf(1));
+        when(dataSetQueryHelperMock.getColumnLongValue(dataSetMock, COLUMN_PROCESSINSTANCEID, 0)).thenReturn(Long.valueOf(1));
+        when(dataSetQueryHelperMock.getColumnStringValue(dataSetMock, COLUMN_DEPLOYMENTID, 0)).thenReturn("testDeployment");
+        when(dataSetQueryHelperMock.getColumnLongValue(dataSetMock, COLUMN_PARENTID, 0)).thenReturn(Long.valueOf(-1));
+
+
+        when(dataSetTaskVarMock.getRowCount()).thenReturn(2); //two domain variables associated
+        when(dataSetDomainDataQueryHelperMock.getColumnLongValue(dataSetTaskVarMock, COLUMN_TASKID, 0)).thenReturn(Long.valueOf(1));
+        when(dataSetDomainDataQueryHelperMock.getColumnStringValue(dataSetTaskVarMock, COLUMN_TASK_VARIABLE_NAME, 0)).thenReturn("var1");
+        when(dataSetDomainDataQueryHelperMock.getColumnStringValue(dataSetTaskVarMock, COLUMN_TASK_VARIABLE_VALUE, 0)).thenReturn("value1");
+
+        when(dataSetDomainDataQueryHelperMock.getColumnLongValue(dataSetTaskVarMock, COLUMN_TASKID, 1)).thenReturn(Long.valueOf(1));
+        when(dataSetDomainDataQueryHelperMock.getColumnStringValue(dataSetTaskVarMock, COLUMN_TASK_VARIABLE_NAME, 1)).thenReturn("var2");
+        when(dataSetDomainDataQueryHelperMock.getColumnStringValue(dataSetTaskVarMock, COLUMN_TASK_VARIABLE_VALUE, 1)).thenReturn("value2");
+
+        //Mock that actually calls the callbacks
+        doAnswer(new Answer() {
 
+            @Override
+            public Object answer(InvocationOnMock invocation) throws Throwable {
+                ((DataSetReadyCallback) invocation.getArguments()[1]).callback(dataSetMock);
+                return null;
+            }
+        }).when(dataSetQueryHelperMock).lookupDataSet(anyInt(), (DataSetReadyCallback) anyObject());
```

Comment:
A better way to define the callback matcher would be to use: `any(DataSetReadyCallback.class)` instead of `(DataSetReadyCallback) anyObject()`